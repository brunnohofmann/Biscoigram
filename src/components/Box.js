import React from 'react';
import { StyleSheet, View} from 'react-native';
import {horizontalPaddingBase} from '../constants/layout';

export default ({children, style}) => (
    <View style={{...styles.box, ...style}}>
        { children }
    </View>
);

const styles = StyleSheet.create({
    box: {

        paddingHorizontal: horizontalPaddingBase,
    },
});
