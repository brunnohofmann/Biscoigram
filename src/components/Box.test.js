import React from 'react';
import {shallow} from 'enzyme';
import Box from './Box';


describe('Box.js test', () => {

    test('Should match with snapshot', () => {
        const boxShallow = shallow(<Box />);
        expect(boxShallow).toMatchSnapshot();
    });


    test('Should match with snapshot', () => {
        const boxShallow = shallow(<Box />);
        expect(boxShallow).toMatchSnapshot();
    });
});
