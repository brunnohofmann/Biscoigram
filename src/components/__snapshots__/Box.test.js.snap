// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box.js test Should match with snapshot 1`] = `
<View
  style={
    Object {
      "paddingHorizontal": 8,
    }
  }
/>
`;
